### ISSUE TITLE
Default content on V3 home page install needs updating

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#3678]
# Problem

- The page is using outdated class names that we have deprecated 
- Events, Articles, and Banners need some updates as well.

# Proposed Solution

- Update all class names to use updated UIDS class names instead of legacy
- Update Events and Articles to use built-in headline field
- https://github.com/uiowa/uiowa/blob/main/docroot/profiles/custom/sitenow/content/block_content/582328d0-e1f0-4de6-843d-e2372c91e6e6.yml needs the layout_builder_style_headline_size = headline_medium

**Comment by bspeare** [comment]
As a part of this work, we might want to consider updating the the home page install to be a recommended home page design and move the block variant documentation/showcase to a page on https://webcommunity.sites.uiowa.edu/ which can be targeted with visual regression testing.

**Comment by joewhitsitt** [comment]
Postponed until https://github.com/uiowa/uiowa/issues/3501 is done

**Comment by richardbporter** [comment]
> As a part of this work, we might want to consider updating the the home page install to be a recommended home page design and move the block variant documentation/showcase to a page on https://webcommunity.sites.uiowa.edu/ which can be targeted with visual regression testing.

Just want to clarify that we don't want to target production sites for visual regression testing but rather what is available in the PR, which right now is default site with default content.

**Comment by richardbporter** [comment]
> Postponed until #3501 is done

There may be some advantages to doing this issue along with #3713 in #4122. If that were to happen, we'd have to redo the articles piece of it after #3501 was complete? Is that accurate?

**Comment by joewhitsitt** [comment]
>we'd have to redo the articles piece of it after #3501 was complete? Is that accurate?

yes
